internal/core/legacy/dao/application.go

```diff
@@ -131,7 +131,9 @@ func (client *DBClient) GetApplicationsByIDs(orgID *int64, projectID *int64, app
 		db = db.Order(fmt.Sprintf("%s", request.OrderBy))
 	}
 	if request.Query != "" {
-		db = db.Where("name LIKE ? OR display_name LIKE ?", strutil.Concat("%", request.Query, "%"), strutil.Concat("%", request.Query, "%"))
+		db = db.Where("name LIKE ? OR display_name LIKE ?", strutil.Concat("%", request.Query, "%"), strutil.Concat("%", request.Query, "%")).
+			Order(gorm.Expr("CHAR_LENGTH(display_name)")).
```

Comment:
Please describe the compatibility of this expression in SQLite.

Reply:
done